### ISSUE TITLE
Null-safety support

### ISSUE THREAD

**Issue by m0nac0** [andrea689/mapbox-gl-dart#12]
... would be great.

There's an automated tool to guide the process, see https://dart.dev/null-safety/migration-guide#migration-tool.

This is required for dependent packages to support null-safety. (https://github.com/tobrun/flutter-mapbox-gl/issues/564)

**Comment by andrea689** [comment]
@m0nac0 thank you, I will try to use this tool as soon as possible

**Comment by andrea689** [comment]
I added NNBD in #13 and my examples works, but maybe some methods used by `flutter-mapbox-gl` need to be fixed.

Until I try and fix any problems, you can try using this branch and report me any problems you find.

```yml
mapbox_gl_dart:
  git:
    url: git@github.com:andrea689/mapbox-gl-dart.git
    ref: null-safety
```

**Comment by kevin-lot** [comment]
@andrea689 Thank you very much !

**Comment by Ichchhie** [comment]
Hi, any updates on this issue or any way we can contribute to this? (https://github.com/tobrun/flutter-mapbox-gl/issues/564)

**Comment by andrea689** [comment]
Hi @Ichchhie, I just released the 0.2.0-nullsafety.0 prerelease.

I don't have time to migrate flutter-mapbox-gl at the moment, but now you can start working on it. 

If you find any problems with this prerelease feel free to open an issue and I'll try to solve it, or you can make a PR.
